cli/embedded-app/package-lock.json

```diff
@@ -0,0 +1,11345 @@
+{
+  "name": "wrapper-app",
+  "version": "0.1.0",
+  "lockfileVersion": 1,
+  "requires": true,
+  "dependencies": {
+    "@babel/code-frame": {
+      "version": "7.0.0",
+      "resolved": "https://purecloud.jfrog.io/purecloud/api/npm/inin-internal-npm/@babel/code-frame/-/code-frame-7.0.0.tgz",
```

Comment:
I can never remember if we decided to leave the internal npm links or not...

Reply:
Ugh, we decided not to, and I added an .npmrc file to try to avoid it, but of course it doesn't impact the nested project. I'll fix that. Good catch.